README.md

```diff
@@ -98,27 +102,30 @@ We don't advise you to use the same EFS instance for persisting the data from mu
 
 ### 2. Add the Alfresco Helm repository to helm
 
-```helm repo add alfresco-incubator https://kubernetes-charts.alfresco.com/incubator```
+```helm repo add alfresco-incubator https://kubernetes-charts.alfresco.com/incubator
```

Comment:
It's a bit more readable when the hashes are on their own line.

Reply:
Done. Thank you!